javascript/javascript2/week1/SpiritNameApp.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <div class="main">
+        <h1>Spirit animal name generator</h1>
+        <h2>Please enter your name:</h2>
+        <input type="text">
+        <button type="button" class="getName">Get a spirit name</button>
+        <button type="button" class="getNewName">Get a new spirit name</button>
+    </div>
+    <div class="outPut">Your spirit name appears here</div>
+    <script>
+        const spiritAnimals = ["Howling Wolf", 
+        "Roaring Lion",
+        "Crouching Tiger",
+        "Crazy Frog",
+        "Flying Fish",
+        "Jumping Horse",
+        "Swimming Duck",
+        "Barking Dog",
+        "Running Cat",
+        "Sneaky Mouse"];
+        const randomSpirit = spiritAnimals[Math.floor(Math.random() * spiritAnimals.length)];
```

Comment:
Hi Younes. Thank you for you review and session. I want to keep it, because i want my first button to work only once, when user got his name. Otherwise my first button will work exactly the same like second.